### PR TITLE
Add rails 3 compatibility

### PR DESCRIPTION
**Pull request by gjhenrique**
#26 

Do you think it's better to merge it in master or in a separate branch?

### CODE REVIEW

File: lib/influxer/metrics/metrics.rb [hunk 1]
```diff
@@ -122,13 +121,18 @@ def quote(name)
       # rubocop:enable Metrics/MethodLength
     end
 
-    delegate :tag_names, to: :class
+    delegate :tag_names, to: 'self.class'
```
Comment (palkan) [inline lib/influxer/metrics/metrics.rb]: Let's get rid of delegation in favour of explicit method:

```ruby
def tag_names
  self.class.tag_names
end
```

That would not depend on Rails at all.

File: lib/influxer/metrics/metrics.rb [hunk 2]
```diff
@@ -122,13 +121,18 @@ def quote(name)
...
     attr_accessor :timestamp
 
     def initialize(attributes = {})
       @attributes = {}
       @persisted = false
-      super
+
+      return unless attributes
```
Comment (palkan) [inline lib/influxer/metrics/metrics.rb]: We can leave `super` here and extract this code into a module to be prepended:

```ruby
prepend(Rails3::Metrics) if RAILS_VERSION < 4
```

File: lib/influxer/metrics/metrics.rb (hunk 1 above)
Comment (gjhenrique) [inline lib/influxer/metrics/metrics.rb]: Good idea.

File: lib/influxer/metrics/metrics.rb (hunk 2 above)
Comment (gjhenrique) [inline lib/influxer/metrics/metrics.rb]: It makes sense...Will work on that

**Comment by palkan** [comment]
Great work! Thanks!

Released as 1.0.0.

File: lib/influxer/metrics/active_model3/model.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+module Influxer
```
Comment (gjhenrique) [inline lib/influxer/metrics/active_model3/model.rb]: wat?

**Comment by gjhenrique** [comment]
Wow.
Thank you for making Influxer =)

**Pull request merged**